From fe11f15cfd0518aa8c054d3794313c22a706961f Mon Sep 17 00:00:00 2001
From: Eric Naim <dnaim@cachyos.org>
Date: Sat, 16 Aug 2025 11:25:26 +0700
Subject: [PATCH 56/56] Revert "hid: Makefile: Add asus-hid-ally to resolve
 conflict"

This reverts commit 2ecf32c3eb31cbb62db9ea3c5ec1b2c3cb6c88a2.

Signed-off-by: Eric Naim <dnaim@cachyos.org>
---
 drivers/hid/Makefile | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/hid/Makefile b/drivers/hid/Makefile
index a0f584754..107cb4b3d 100644
--- a/drivers/hid/Makefile
+++ b/drivers/hid/Makefile
@@ -172,8 +172,6 @@ obj-$(CONFIG_INTEL_ISH_HID)	+= intel-ish-hid/
 
 obj-$(CONFIG_AMD_SFH_HID)       += amd-sfh-hid/
 
-obj-$(CONFIG_ASUS_ALLY_HID)  += asus-ally-hid/
-
 obj-$(CONFIG_SURFACE_HID_CORE)  += surface-hid/
 
 obj-$(CONFIG_INTEL_THC_HID)     += intel-thc-hid/
-- 
2.50.1

